apps/condo/domains/user/schema/ForgotPasswordAction.js

```diff
@@ -0,0 +1,208 @@
+const isEmpty = require('lodash/isEmpty')
+
+
+const ForgotPasswordAction = new GQLListSchema('ForgotPasswordAction', {
+    schemaDoc: 'Forgot password actions is used for anonymous user password recovery procedure',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+        user: {
+            schemaDoc: 'Ref to the user. The object will be deleted if the user ceases to exist',
+            type: Relationship,
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+        token: {
+            schemaDoc: 'Unique token to complete confirmation',
+            type: Text,
+            isRequired: true,
+        },
+        requestedAt: {
+            schemaDoc: 'DateTime when confirm phone action was started',
+            type: DateTimeUtc,
+            isRequired: true,
+        },
+        expiresAt: {
+            schemaDoc: 'When password recovery action becomes invalid',
+            type: DateTimeUtc,
+            isRequired: true,
+        },
+        usedAt: {
+            schemaDoc: 'When password recovery action was completed',
+            type: DateTimeUtc,
+            isRequired: false,
+        },
+    },
+    plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
+    access: {
+        read: access.canReadForgotPasswordAction,
+        create: access.canManageForgotPasswordAction,
+        update: access.canManageForgotPasswordAction,
+        delete: false,
+        auth: true,
+    },
+})
+
+const ForgotPasswordService = new GQLCustomSchema('ForgotPasswordService', {
+    types: [
+        {
+            access: true,
+            type: 'input CheckPasswordRecoveryTokenInput { token: String! }',
+        },
+        {
+            access: true,
+            type: 'type CheckPasswordRecoveryTokenOutput { status: String! }',
+        },
+        {
+            access: true,
+            type: 'input StartPasswordRecoveryInput { email: String!, sender: JSON!, dv: Int! }',
+        },
+        {
+            access: true,
+            type: 'type StartPasswordRecoveryOutput { status: String! }',
+        },
+        {
+            access: true,
+            type: 'input ChangePasswordWithTokenInput { token: String!, password: String! }',
+        },
+        {
+            access: true,
+            type: 'type ChangePasswordWithTokenOutput { status: String! }',
+        },
+
+    ],
+    queries: [
+        {
+            access: true,
+            schema: 'checkPasswordRecoveryToken(data: CheckPasswordRecoveryTokenInput!): CheckPasswordRecoveryTokenOutput',
+            resolver: async (parent, args, context, info, extra) => {
+                const { data: { token } } = args
+                const now = extra.extraNow || Date.now()
+                const actions = await ForgotPasswordActionGQL.getAll(context, {
```

Comment:
It's better to use ForgotPasswordActionUtil because we use usually ForgotPasswordActionGQL for /gql.js imports